gradle/libs.versions.toml

```diff
@@ -3,8 +3,8 @@
 java-compilation = "21"
 # The java-target version is the lowest supported LTS version of Java. Jar's produced are bytecode compatible with this version.
 java-target = "8"
-kotlin = "1.9.24"
-kotlinDev = "1.9.24"
+kotlin = "2.0.0"
+kotlinDev = "2.0.0"
```

Comment:
https://github.com/JetBrains/kotlin/releases/tag/v2.0.20-Beta1

Reply:
Technically you're right that we can update this. But honestly, I do not have the idea that anyone is using a `snapshot` version, let alone the `dev-snapshot` version. Recently it was broken for several weeks, without anybody noticing it.